docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md

```diff
@@ -0,0 +1,136 @@
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+* User has deployed Kubernetes using Cluster API and has it connected as Azure Arc enabled Kubernetes cluster.
+
+* User is editing the environment variables on the Shell script file (1-time edit) which then be used throughout the extension deployment. 
+
+* User is running the shell script. The script will use the extension management feature of Azure Arc to deploy the Azure Defender extension on the Azure Arc connected cluster.
+
+* User is veryfing the cluster is shown in Azure Defender and that the extension is deployed.
+
+* User is simulating a security alert on the Kubernetes cluster that will cause Azure Defender to detect this activity and trigger a security alert.
+
+## Create Azure Defender extensions instance
+
+To create a new extension Instance, we will use the _k8s-extension create_ command while passing in values for the mandatory parameters. This scenario provides you with the automation to deploy the Azure Defender extension on your Azure Arc enabled Kubernetes cluster.
+
+* Before integrating the cluster with Azure Defender, click on the "Security (preview)" blade for the connected Arc cluster to show how the cluster is not currently being assessed by Azure Defender.
+
+    ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource](./01.png)
+
+    ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource extensions](./02.png)
+
+* Edit the environment variables in the script to match your environment parameters followed by running the ```. ./azure_defender_k8s_extension.sh``` command.
+
+    ![Screenshot parameter examples](./03.png)
+
+    > **Note: The extra dot is due to the shell script has an *export* function and needs to have the vars exported in the same shell session as the rest of the commands.**
+
+   The script will:
+
+  * Login to your Azure subscription using the SPN credentials
+  * Add or Update your local _connectedk8s_ and _k8s-extension_ Azure CLI extensions
+  * Create Azure Defender k8s extension instance
+
+* You can see that Azure Defender is enabled once you visit the security tab section of the Azure Arc enabled Kubernetes cluster resource in Azure.
+
+![Screenshot extension deployment security tab](./04.png)
+
+* Also verify under the Extensions section of the Azure Arc enabled Kubernetes cluster that the Azure Defender extension is correctly installed.
+
+![Screenshot extension deployment](./05.png)
+
+* You can also verify the deployment by running the command below:
+
+```bash
+kubectl get pods -n azuredefender --kubeconfig $CAPI_WORKLOAD_CLUSTER_NAME.kubeconfig
```

Comment:
```suggestion
kubectl get pods -n azuredefender --kubeconfig $CAPI_WORKLOAD_CLUSTER_NAME.kubeconfig
```

Outside the script, the environment var will not work since it's in a different shell environment.

Reply:
changed to <cluster-name>.kubeconfig